coderd/prometheusmetrics/prometheusmetrics.go

```diff
@@ -516,6 +516,31 @@ func AgentStats(ctx context.Context, logger slog.Logger, registerer prometheus.R
 	}, nil
 }
 
+// Experiments registers a metric which indicates whether each experiment is enabled or not.
+func Experiments(_ slog.Logger, registerer prometheus.Registerer, exps []string, all codersdk.Experiments) error {
```

Comment:
I wanted to be able to inject it because it makes testing easier, and there's no need for an explicit dependency on that sentinel value from another package.

Reply:
I see 👍 Frankly speaking, I wouldn't be concerned about this dependency, but rather prefer having a test that guarantees the right set of metrics being exposed (ALL).